cfgov/jinja2/v1/owning-a-home/_templates/ratings-form.html

```diff
@@ -1,9 +1,10 @@
 <form method="post"
-      class="o-form
-             oah-ratings-form
+      class="oah-ratings-form
+             o-form
              block
              block__bg
-             block__border">
+             block__border
+             u-mb15">
```

Comment:
This makes the bottom padding consistent across https://www.consumerfinance.gov/owning-a-home/closing-disclosure/ and https://www.consumerfinance.gov/owning-a-home/loan-estimate/. If we wanted a larger gap we could remove this.